### PR TITLE
MAP-30 Add Create Source page

### PR DESCRIPTION
**Pull request by PawelGutkowski**
https://issues.openmrs.org/browse/MAP-30

### CODE REVIEW

File: owa/app/js/home/home.html [hunk 1]
```diff
@@ -1,4 +1,9 @@
+<div>
+    <a class="button" href="#/source/add"><i class="icon-plus"></i>&nbsp;Add new metadata source</a>
```
Comment (dkayiwa) [inline owa/app/js/home/home.html]: How about localising the above text?

File: owa/app/js/createsource/createsource.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<h1><i class="icon-pencil"></i>&#160;Create Metadata Source</h1>
+<form name="createSource">
+    <div align="center">
+        <fieldset style="width: 90%">
+            <p class="input-position-class">
```
Comment (dkayiwa) [inline owa/app/js/createsource/createsource.html]: Can we localise the text in this file?

**Comment by PawelGutkowski** [comment]
@dkayiwa I've adressed your comments and introduced translations module from `openmrs-contrib-uicommons`, see screenshots:
screenshot from 2016-10-03 10 56 52
screenshot from 2016-10-03 10 57 23
screenshot from 2016-10-03 10 59 39

File: owa/app/js/createsource/createsource.html (hunk 2 above)
Comment (PawelGutkowski) [inline owa/app/js/createsource/createsource.html]: see https://github.com/openmrs/openmrs-module-metadatamapping/pull/38#issuecomment-251058690

File: owa/app/js/home/home.html (hunk 1 above)
Comment (PawelGutkowski) [inline owa/app/js/home/home.html]: see https://github.com/openmrs/openmrs-module-metadatamapping/pull/38#issuecomment-251058690

**Comment by dkayiwa** [comment]
@rkorytkowski do you think this is ready for merging?

**Pull request merged**